cob_hardware_config/robots/cob4-16/config/arm_left_driver.yaml

```diff
@@ -14,16 +14,16 @@ defaults:
 
 nodes:
   - name: arm_left_1_joint
-    id: 61
+    id: 51
```

Comment:
same comment as above: why switch arm left and right?

Reply:
this is how the robot hw is set up...